scripts/tests/java/discover_test.py

```diff
@@ -0,0 +1,71 @@
+#!/usr/bin/env python3
+
+#
+#    Copyright (c) 2022 Project CHIP Authors
+#    All rights reserved.
+#
+#    Licensed under the Apache License, Version 2.0 (the "License");
+#    you may not use this file except in compliance with the License.
+#    You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS,
+#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    See the License for the specific language governing permissions and
+#    limitations under the License.
+#
+
+import logging
+import os
+import sys
+import asyncio
+import queue
+import subprocess
+import threading
+import typing
+import argparse
+from colorama import Fore, Style
+from java.base import DumpProgramOutputToQueue
+
+
+class DiscoverTest:
+    def __init__(self, thread_list: typing.List[threading.Thread], queue: queue.Queue, cmd: [], args: str):
+        self.thread_list = thread_list
+        self.queue = queue
+        self.command = cmd
+
+        parser = argparse.ArgumentParser(description='Process discover arguments.')
+
+        parser.add_argument('command', help="Command name")
+        parser.add_argument('-n', '--nodeid', help="DNS-SD name corresponding with the given node ID", default='1')
+        parser.add_argument('-f', '--fabricid', help="DNS-SD name corresponding with the given fabric ID", default='1')
+        parser.add_argument('-p', '--paa-trust-store-path', dest='paa_trust_store_path',
+                            help="Path that contains valid and trusted PAA Root Certificates")
+
+        args = parser.parse_args(args.split())
+
+        self.command_name = args.command
```

Comment:
These arguments don't seem to be used. Are they for the future use?

Reply:
Exactly, those arguments are aligned with chip-tool, and will be used in 'discover resolve' command.